File: test/qr.clj
(import '[com.google.zxing BarcodeFormat]
        '[com.google.zxing.qrcode QRCodeWriter])

(let [writer (QRCodeWriter.)
      matrix (.encode writer "unsandbox-qr-ok" BarcodeFormat/QR_CODE 0 0)
      rows (.getHeight matrix)]
  (println (str "QR:unsandbox-qr-ok:ROWS:" rows)))

File: test/qr.pl
use Imager::QRCode;
my $qr = Imager::QRCode->new(size => 1, margin => 0);
my $img = $qr->plot("unsandbox-qr-ok");
my $rows = $img->getheight();
print "QR:unsandbox-qr-ok:ROWS:$rows\n";
